sites/README.md

```diff
@@ -0,0 +1,293 @@
+# Valid properties
+
+List of possible values that can be used
+
+## name
+**(Mandatory)**
+
+**Type:** String
+
+This property will be used mainly for all the Kubernetes metadata name, it will also be used to set some properties as a default value if they have not been defined as tlsSecretName.
+
+Example:
+```
+name: canonical-com
+```
+
+## domain
+**(Mandatory)**
+
+**Type:** String
+
+This property define the project domain, it will be used for the Service, Deployment and Ingress objects.
+
+Example:
+```
+domain: canonical.com
+```
+
+## noIngress
+
+**Type:** Boolean
+
+**Default:** False
+
+When this property is set to true, the Ingress object will be empty.
+
+Example:
+```
+noIngress: true
+```
+
+## container
+**(Mandatory)**
+
+This property is for container related configurations. Here you can set the [image](#image) and the [tag](#tag).
+
+### image
+**(Mandatory)**
+
+**Type:** String
+
+Set the Docker image you want to use.
+
+Example:
+```
+container:
+    image: prod-comms.docker-registry.canonical.com/canonical.com
+```
+
+### tag
+
+**Type:** String
+
+**Default:** latest
+
+You can specify the tag you want to use with the image.
+
+Example:
+```
+container:
+    image: prod-comms.docker-registry.canonical.com/canonical.com
+    tag: a264efb326485
+```
+
+## wwwPrefixHost
+
+**Type:** Boolean
+
+**Default:** True
+
+When True it will include an extra host with the www prefix to the Ingress object.
+
+Example:
+```
+wwwPrefixHost: false
+```
+
+## tlsSecretName
+
+**Type:** String
+
+**Default:** {name}-tls
+
+Here you can define the name of the TLS secret for the domain. This name will be used for the www prefix host if `wwwPrefixHost` is True.
+
+Example:
+```
+tlsSecretName: cloud-init-org-tls
+```
+
+## useSquid
+
+**Type:** Boolean
+
+**Default:** False
+
+This will use our ConfigMap object "proxy-config", you can take a look to this object here: `configmaps/proxy-config.yaml`
+
+Example:
+```
+useSquid: True
+```
+
+## readinessPath
+
+**Type:** String
+
+**Default:** /_status/check
+
+Define the URL path to do a readiness probe.
+
+Example:
+```
+readinessPath: /
+```
+
+## readinessPeriodSeconds
+
+**Type:** Int
+
+**Default:** 5
+
+How often (in seconds) to perform the probe. Default to 5 seconds. Minimum value is 1.
+
+Example:
+```
+readinessPeriodSeconds: 10
+```
+
+## readinessSuccessThreshold
+
+**Type:** Int
+
+**Default:** 1
+
+Minimum consecutive successes for the probe to be considered successful after having failed. Defaults to 1. Must be 1 for liveness. Minimum value is 1.
+
+Example:
+```
+readinessSuccessThreshold: 5
+```
+
+## readinessTimeoutSeconds
+
+**Type:** Int
+
+**Default:** 3
+
+Number of seconds after which the probe times out. Defaults to 3 second. Minimum value is 1.
+
+Example:
+```
+readinessTimeoutSeconds: 5
+```
+
+## memory
+
+**Type:** String
+
+**Default:** 256Mi
+
+It will define the container memory limit.
+
+Example:
+```
+memory: 512Mi
+```
+
+## envSecrets
```

Comment:
This should just be called `env`, or `environment_variables`. These are explicitly *not* secrets, as their values are included in this repository in plain text for anyone to read.

Reply:
Renamed to env